docs/developer/front-end/3.1-creating-themes.md

```diff
@@ -0,0 +1,345 @@
+# Creating Themes and Subthemes
+
+In Drupal 10 you can create base themes and subthemes. A base theme provides some set of predefined styles and functionality that a frontend developer can base a subtheme upon. Any theme can be used as a base theme, even if it is a subtheme of some other theme.
+
+## Core Themes
+
+Drupal 10 comes with several preloaded themes, located in `core/themes/`.
+
+1. **Olivero**: Olivero is a core theme and the new default theme for Drupal starting with version 9.4.0.
+2. **Claro**: A clean, accessible and powerful administration theme based on the Drupal Design System.
+3. **Stark**: Is not based on any other theme. It is intentionally styleless to demonstrate default Drupal HTML, CSS and Javacript from core and contrib modules.
+4. **Stable9**: Stable 9 is intended to be used as a base theme if you want to begin with minimal markup and styling **and** you need assurance that your theme is not changed in any way by a Drupal update.
+
+In addition, there is a [command line tool](https://www.drupal.org/docs/core-modules-and-themes/core-themes/starterkit-theme) called "Starterkit" that allows theme developers to easily start a new theme.
+
```

Comment:
Add a line or two to emphasize the deprecation of Stable and the importance of transitioning to Starterkit for creating themes, as this reflects the forward-facing best practices in the Drupal community.